Gracefully degradate AntTarget task (#33817)
gradle/gradle: platforms/documentation/docs/src/docs/userguide/authoring-builds/other/ant.adoc

WARNING: Ant integration is not fully compatible with the <<configuration_cache.adoc#config_cache,configuration cache>>.
Using link:{javadocPath}/org/gradle/api/Task.html#getAnt--[Task.ant] to run Ant task in the task action may work, but importing the Ant build is not supported.

> Due to that configuration cache is automatically disabled when importing the Ant build.

Reviewer: ```suggestion
As a result, the configuration cache is automatically disabled when importing an Ant build.
```
